### DISCUSSION TITLE
Behavior of ${...} changed?

### DISCUSSION THREAD

**Discussion by tshead2** [xonsh/xonsh#4424] [Q&A] [answered]
Before 0.10, the command-line expression

    ${...}

printed out all my environment variables.  Now, it prints the object:

    xonsh.__amalgam__.Env(...)

I'm curious why things changed, and whether there's a better way to list variables ... what I've got so far:

    sorted(${...}) # Just the keys
    sorted(${...}.items()} # Keys and values

Many thanks,
Tim

**Comment by gforsyth** [comment, marked as answer]
Hey @tshead2 -- this is a regression.  I have a quick PR to fix this up.  Thanks for raising the issue!

**Reply by gforsyth** [reply]
blarg, got blocked by corporate proxy, give me a few

**Reply by tshead2** [reply]
@gforsyth - Excellent, thanks for the quick response.

**Reply by gforsyth** [reply]
You can check out https://github.com/xonsh/xonsh/pull/4427 for the fix
